File: python_master_reference/05_strings.py
"""
05 - Strings
String means text.
"""

first_name = "Khadir"

last_name = "Zavareh"

full_name = first_name + " " + last_name  # Join strings together

print(full_name)

message = f"My name is {first_name}."  # f-string puts variables inside text

print(message)

sentence = "Python is useful for test automation."

print(sentence.upper())  # Uppercase

print(sentence.lower())  # Lowercase

print(sentence.replace("Python", "Robot Framework"))  # Replace text

print(len(sentence))  # Count characters

print(sentence[5])  # First character

print(sentence[0:6])  # Characters from index 0 to 5

File: typescript_master_reference/05_strings.ts
/*
05 - Strings
String means text.
*/
const firstName = "Khadir";
const lastName = "Zavareh";
const fullName = firstName + " " + lastName; // Join strings
console.log(fullName);
const message = `My name is ${firstName}.`; // Template literal
console.log(message);
const sentence = "TypeScript is useful for test automation.";
console.log(sentence.toUpperCase()); // Uppercase
console.log(sentence.toLowerCase()); // Lowercase
console.log(sentence.replace("TypeScript", "Playwright")); // Replace text
console.log(sentence.length); // Count characters
console.log(sentence[0]); // First character
console.log(sentence.slice(0, 10)); // Characters from index 0 to 9
